$OpenBSD: patch-Tests_CMakeOnly_AllFindModules_CMakeLists_txt,v 1.2 2012/06/15 10:26:56 dcoppa Exp $

Add FindLibLZMA Module
(upstream git commit ee26019bd433020b8aad3c3a70bcb537e03b04ba)

FindPkgConfig.cmake: fix documented output variable not set
(upstream git commit 3ea850a5023060b84dcc0e6f0098c32c28b15807)

--- Tests/CMakeOnly/AllFindModules/CMakeLists.txt.orig	Wed Apr 18 20:10:54 2012
+++ Tests/CMakeOnly/AllFindModules/CMakeLists.txt	Fri Jun 15 11:47:17 2012
@@ -71,7 +71,7 @@ endmacro(check_version_string)
 # reported.
 
 foreach(VTEST ALSA ARMADILLO BZIP2 CUPS CURL EXPAT FREETYPE GETTEXT GIT HSPELL
-        JASPER LIBXML2 LIBXSLT PERL PostgreSQL TIFF ZLIB)
+        JASPER LIBLZMA LIBXML2 LIBXSLT PERL PKG_CONFIG PostgreSQL TIFF ZLIB)
     check_version_string(${VTEST} ${VTEST}_VERSION_STRING)
 endforeach(VTEST)
 
@@ -82,4 +82,3 @@ endforeach(VTEST)
 
 check_version_string(PYTHONINTERP PYTHON_VERSION_STRING)
 check_version_string(SUBVERSION Subversion_VERSION_SVN)
-check_version_string(PKGCONFIG PKG_CONFIG_VERSION_STRING)
